interface/main/left_nav.php

```diff
@@ -1047,6 +1048,7 @@ function selpopup(selobj) {
   <?php if (!$GLOBALS['disable_pat_trkr'] && !$GLOBALS['disable_calendar']) genTreeLink('RTop','pfb',xl('Flow Board')); ?>
   <?php genTreeLink('RBot','msg',xl('Messages')); ?> 
   <?php if($GLOBALS['portal_offsite_enable'] && $GLOBALS['portal_offsite_address'] && acl_check('patientportal','portal'))  genTreeLink('RTop','app',xl('Portal Activity')); ?>
+  <?php if($GLOBALS['portal_onsite_enable'] && acl_check('patientportal','portal'))  genTreeLink('RTop','aop',xl('Onsite Dashboard')); ?>
```

Comment:
I've changed so many times I lost track. Personally, I like Portal Dashboard but I think you are going for consistency.